zeebe/gateway-protocol/src/main/proto/rest-api.yaml

```diff
@@ -3960,6 +3960,13 @@ components:
         licenseType:
           description: Will return the license type property of the Camunda license
           type: string
+        isCommercial:
+          description: Will be false when a license contains a non-commerical=true property
+          type: boolean
+        expiresAt:
+          description: The time in UTC when the Camunda license expires
```

Comment:
🔧 Tying this to UTC by definition is not needed and makes it a breaking change if we ever return anything else. The offset date time that will be returned here contains offset information. This will be sufficient for any consuming application already.